lib/tasks/move_banner_images.rake

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+# @see https://github.com/scientist-softserv/palni-palci/issues/674
+namespace :hyku do
+  desc 'Move all banner images to new expected location'
+  task copy_banner_images: :environment do
+    files_in_old_location = `find #{Rails.root}/public/uploads/*/site/banner_image/* -type f`.split("\n")
+    new_location = Rails.root.join('public', 'system', 'banner_images', '1', 'original')
```

Comment:
Non-blocking, simply curious: what's the significance of the "1" directory?

Reply:
That is the `Site` ID. Every tenant is represented by an `Account` object. Each `Account` has one `Site`. Since the `Site` db table is within the tenant-specific db schema, _every_ `Site` ID is guaranteed to be `1`.

Reply:
I'll add a code comment explaining that.